tests/serve_app/oxo_test.py

```diff
@@ -459,3 +459,59 @@ def testDeleteScanMutation_whenScanDoesNotExist_returnErrorMessage(
 
     assert response.status_code == 200, response.get_json()
     assert response.get_json()["errors"][0]["message"] == "Scan not found."
+
+
+def testStopScanMutation_always_shouldStopScan(
```

Comment:
```suggestion
def testStopScanMutation_whenScanIsRunning_shouldStopScan(
```

The second test has a specific condition. so always is not useful in this case.

Reply:
make sense